src/theme/misc.js

```diff
@@ -1,7 +1,7 @@
 import { colors } from './colors';
 
 export const overlayIconSize = '300px';
-export const textColor = colors.black;
+export const textColor = colors.white;
```

Comment:
This seems like a fairly large change? The snapshot above suggests that the `Text` component now has `color: hsl(0,0,100);` which seems like an issue?

Reply:
My mistake! Didn’t realise how big of a change that would be.